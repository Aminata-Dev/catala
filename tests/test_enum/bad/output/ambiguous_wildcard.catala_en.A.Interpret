[ERROR] Couldn't infer the enumeration name from lonely wildcard (wildcard cannot be used as single match case)

  --> test_enum/bad/ambiguous_wildcard.catala_en
   | 
15 |     -- anything : 31
   |        ^^^^^^^^^^^^^
   + Wildcard cannot be used as single match case
