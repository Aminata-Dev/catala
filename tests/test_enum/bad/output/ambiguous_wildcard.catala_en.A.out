[ERROR] Couldn't infer the enumeration name from lonely wildcard. (Wildcard cannot be used as single match case)
[ERROR] 
[ERROR]   --> test_enum/bad/ambiguous_wildcard.catala_en
[ERROR]    | 
[ERROR] 15 |     -- Anything : 31
[ERROR]    |        ^^^^^^^^^^^^^
[ERROR]    + Wildcard cannot be used as single match case
